packages/analyzer/src/model/class.ts

```diff
@@ -128,6 +131,7 @@ export interface ClassInit {
   readonly properties?: ImmutableMap<string, Property>;
   readonly staticMethods: ImmutableMap<string, Method>;
   readonly methods?: ImmutableMap<string, Method>;
+  readonly constructorMethod?: Method;
```

Comment:
I think `constructor` is a better name here, rather than `constructorMethod`

Reply:
But `constructor` is a name that is special, as you can do stuff like `Function.constructor` to get the actual object. I did think of it, but decided to append `Method` to it to prevent this clash. Should I still do it?